set.mm

```diff
@@ -794993,6 +795171,466 @@ regarding the multiplication ( ~ rngqiprnglin ) via ~ rngqiprnglinlem1
+      syl2anb wb cbs issubrng2 ax-mp mpbir2an ) BAUCIJZBAUDIJZEKZFKZAUMIZLZBJZF
+      BUAEBUAZABCDUEWREFBBWNBJWNGKZMNZOZGPUBZWOHKZMNZOZHPUBZWRWOBJGABWNCDUFHABW
+      OCDUFXCXGWRXBXGWRUGGPWTPJZXGXBWRXHXFXBWRUGHPXHXDPJZQZXFXBWRXJXFXBQZQZXAXE
+      WPLZPMUHZUIZWQBXJXMXOJXKXJXMWTXDRUMIZLZMMXPLZNXOXJWTMXDMRRWPAXPXPUJUJPPCU
+      KUKRUJJXJULSZXSXHXIUNXJUOZXHXIUPXTXJXQWTXDTLPXPTWTXDTXPUQURZUSWTXDUTVAZXR
+      PJXJXRMPXRMMTLMXPTMMYAUSMVDVBVCZVEVFSXPVGZYDWPVGZVHXJXQXRPXNYBXJXRMXNYCMX
+      NJXJMVIVJSVAVKVLVMXKWQXMOZXJXBXFYFWNXAWOXEWPVNVOVPBXOOXLDSVQVRVSVTWAWBWFW
+      CAWDJWLWMWSQWGACWEEFBAWHIZAWPYGVGYEWIWJWK $.
+
+    $d J a $.  $d X a $.
+    pzriprng.j $e |- J = ( R |`s I ) $.
+    $( Lemma 6 for ~ pzriprng : ` J ` has a ring unity.  (Contributed by AV,
+       19-Mar-2025.) $)
+    pzriprnglem6 $p |- ( X e. I -> ( ( <. 1 , 0 >. ( .r ` J ) X ) = X
+                                     /\ ( X ( .r ` J ) <. 1 , 0 >. ) = X ) ) $=
+      ( va wcel cc0 cop wceq cz c1 co czring oveqi a1i crg cmul cv cmulr cfv wa
+      wrex pzriprnglem3 csubrng pzriprnglem5 ressmulr eqcomd zringbas zringring
+      eqid ax-mp 1zzd zringmulr zmulcld eqeltrrid eqcomi eqeltri xpsmul mullidd
+      0z zcn eqtr3id 0cn mul02i eqtri opeq12d 3eqtrd mulridd eqtrid jca eqeq12d
+      id oveq2 oveq1 anbi12d syl5ibrcom rexlimiv sylbi ) DBIDHUAZJKZLZHMUENJKZD
+      CUBUCZOZDLZDWEWFOZDLZUDZHABDEFUFWDWKHMWBMIZWKWDWEWCWFOZWCLZWCWEWFOZWCLZUD
+      WLWNWPWLWMWEWCAUBUCZOZNWBPUBUCZOZJJWSOZKWCWMWRLWLWFWQWEWCBAUGUCZIZWFWQLAB
+      EFUHXCWQWFBACWQXBGWQUMZUIUJUNZQRWLNJWBJPPWQAWSWSSSMMEUKUKPSIWLULRZXFWLUOZ
+      JMIZWLVCRZWLVOZXIWLWTNWBTOZMTWSNWBUPQZWLNWBXGXJUQURXAMIWLXAJJTOZMWSTJJTWS
+      UPUSZQZXHXMMIVCXHJJXHVOZXPUQUNUTRZWSUMZXRXDVAWLWTWBXAJWLWTXKWBXLWLWBWBVDZ
+      VBVEXAJLWLXAXMJXOJVFVGVHRZVIVJWLWOWCWEWQOZWBNWSOZXAKWCWOYALWLWFWQWCWEXEQR
+      WLWBJNJPPWQAWSWSSSMMEUKUKXFXFXJXIXGXIWLYBWBNTOZMTWSWBNUPQWLWBNXJXGUQURXQX
+      RXRXDVAWLYBWBXAJWLYBYCWBWSTWBNXNQWLWBXSVKVLXTVIVJVMWDWHWNWJWPWDWGWMDWCDWC
+      WEWFVPWDVOZVNWDWIWODWCDWCWEWFVQYDVNVRVSVTWA $.
+
+    $d J i x $.
+    $( Lemma 7 for ~ pzriprng : ` J ` is a unital ring.  (Contributed by AV,
+       19-Mar-2025.) $)
+    pzriprnglem7 $p |- J e. Ring $=
+      ( vi vx wcel cv cfv co wceq wa wral ax-mp c1 cc0 cz eqid crg crng csubrng
+      cmulr cbs wrex pzriprnglem5 subrngrng cop csn cxp c0ex opelxpii subrngbas
+      1z snid eqtr3i eleqtrri oveq1 eqeq1d ovanraleqv id wi eleq2i pzriprnglem6
+      sylbir a1i ralrimiv rspcedvdw isringrng mpbir2an ) CUAICUBIZGJZHJZCUDKZLZ
+      VNMZVNVMVOLVNMNHCUEKZOZGVRUFZBAUCKIZVLABDEUGZBACFUHPQRUIZVRIZVTWCSRUJZUKZ
+      VRQRSWEUORULUPUMBVRWFWABVRMWBBACFUNPZEUQURWDVSWCVNVOLZVNMZVNWCVOLVNMNZHVR
+      OGWCVRVQWIHVNVMVNVOVRWCVMWCMVPWHVNVMWCVNVOUSUTVAWDVBWDWJHVRVNVRIZWJVCWDWK
+      VNBIWJBVRVNWGVDABCVNDEFVEVFVGVHVIPGHVRCVOVRTVOTVJVK $.
+
+    $d I a b c y x $.  $d R c $.
+    $( Lemma 8 for ~ pzriprng : ` J ` is a two-sided ideal of the non-unital
```

Comment:
```suggestion
    $( Lemma 8 for ~ pzriprng : ` I ` is a two-sided ideal of the non-unital
```

Reply:
According to our definition, ` I ` (the base set of ` J `) is a 2-sided ideal, but in literature ` J ` itself is called 2-sided ideal. Therefore, both variants would be correct. As a compromise, I used both now...